### ISSUE TITLE
"linuxkit" binary is never moved, after build

### ISSUE THREAD

**Issue by vmlemon** [linuxkit/linuxkit#3672]
**Description**

When building LinuxKit, on Darwin/ARM64 (11.2.3 (20D91), the resulting ```linuxkit``` binary is left at ```src/cmd/linuxkit/bin```, instead of being moved into ```bin```:

```
tyson§Tysons-Air linuxkit % file tyson§Tysons-Air linuxkit % file ./src/cmd/linuxkit/bin/linuxkit 
./src/cmd/linuxkit/bin/linuxkit: Mach-O 64-bit executable arm64
tyson§Tysons-Air linuxkit % ls -la bin 
total 12456
drwxr-xr-x   3 tyson  staff       96 May 19 12:50 .
drwxr-xr-x  32 tyson  staff     1024 May 19 12:50 ..
-rwxr-xr-x   1 tyson  staff  6374530 May 19 12:50 rtflinuxkit 
./src/cmd/linuxkit/bin/linuxkit: Mach-O 64-bit executable arm64
tyson§Tysons-Air linuxkit % ls -la bin 
total 12456
drwxr-xr-x   3 tyson  staff       96 May 19 12:50 .
drwxr-xr-x  32 tyson  staff     1024 May 19 12:50 ..
-rwxr-xr-x   1 tyson  staff  6374530 May 19 12:50 rtf
```

This also means that the file is never copied, when doing a ```sudo gmake install```

**Steps to reproduce the issue:**
Check out the repository, using Git, and run ```gmake```, in the repository directory. 

**Describe the results you received:**
Results as in Description.

**Describe the results you expected:**
I would expect the ```linuxkit``` binary to be in the correct location, post-build.

**Additional information you deem important (e.g. issue happens only occasionally):**

**Comment by deitch** [comment]
It never was meant to go to `./src/cmd/linuxkit/bin/linuxkit` but rather to `./bin/linuxkit`. I think this might be a regression caused by #3610 , what do you think @dave-tucker ?

**Comment by dave-tucker** [comment]
Yep this was reported in another issue and there is a fix proposed in #3659 
There is something janky happening with the tar extraction so I want to check it out before I LGTM. I'll try and get to it today.

**Comment by dave-tucker** [comment]
closed by #3659
